aicsimageio/tests/writers/test_rgb_writer.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from pathlib import Path
+
+import dask.array as da
+import numpy as np
+import pytest
+
+from aicsimageio import exceptions
+from aicsimageio.writers import RGBWriter
+
+
+@pytest.mark.parametrize(
+    "array_creator", [lambda shape: np.random.rand(*shape), da.random.random]
+)
+@pytest.mark.parametrize(
+    "shape, dim_order",
+    [
+        ((100, 100, 3), None),
```

Comment:
I think you need to add the possibly very realistic test case where someone passes perfectly good volume data in to the wrong writer.

Reply:
Is this not done by the `(1, 1, 1, 1)` (4D) image check? I can add a `metadata` check that will simply be ignored as a part of `kwargs` though.

Reply:
I see, it just wasn't passing in any dims.  I'd add the dims as a string

Reply:
also a 5d and 6d which could easily come from aicsimage

Reply:
Added tests for 5d and 6d. The 6d test includes a dim order.